Parsers/ASimWebSession/Parsers/ASimWebSessionFortinetFortiGate.yaml

```diff
@@ -0,0 +1,126 @@
+Parser:
+  Title: Web Session ASIM parser for Fortinet FortiGate
```

Comment:
please attach sample logs to test parser

Reply:
Solved as discussed